README.md

```diff
@@ -11,30 +11,29 @@ We have different feature files for unit and integration tests. The unit tests s
 
 # Test Descriptions
 
-| tag                    | description |
-| ---------------------- | ----------- |
-| @algod                 | General tests against algod REST endpoints. |
-| @applications          | Submit all types of application transactions. |
-| @applications.verified | Submit all types of application transactions and verify account state. |
-| @assets                | Submit all types of asset transactions. |
-| @compile               | Test the algod compile endpoint. |
-| @dryrun                | Test the algod dryrun endpoint. |
-| @dryrun.testing        | Test the testing harness that relies on dryrun endpoint. |
-| @indexer               | Test all types of indexer queries and parameters against a static dataset. |
-| @kmd                   | Test the kmd REST endpoints. |
-| @send                  | Test the ability to submit transactions to algod. |
-| @templates             | Integration tests for the TEAL template utilities. |
-| @unit                  | Select all unit tests. |
-| @unit.algod            | Algod REST API unit tests. |
-| @unit.applications     | Application endpoints added to Algod and Indexer. |
-| @unit.dryrun           | Dryrun endpoint added to Algod. |
-| @unit.indexer          | Indexer REST API unit tests. |
-| @unit.indexer.rekey    | Rekey endpoints added to Algod and Indexer |
-| @unit.offline          | The first unit tests we wrote for cucumber. |
-| @unit.transactions     | Transaction golden tests. |
-| @unit.rekey            | Rekey Transaction golden tests. |
-| @unit.responses        | REST Client Response serialization tests. |
-| @unit.tealsign         | Test TEAL signature utilities. |
+| tag                   | description |
```

Comment:
Any reason for removing the header bar? I could take it or leave it, just curious.

Reply:
oops, bad merge!